app/assets/javascripts/actions/article_finder_action.js

```diff
@@ -26,7 +37,7 @@ export const sortArticleFinder = (key) => {
   };
 };
 
-export const fetchCategoryResults = (category, course, cmcontinue = '', continueResults = false) => (dispatch, getState) => {
+export const fetchCategoryResults = (category, home_wiki, cmcontinue = '', continueResults = false) => (dispatch, getState) => {
```

Comment:
Should this just be `wiki` instead of `home_wiki`? I assume the idea is to make it general so that it's still relevant outside the context of a course.